sportsreference/ncaaf/player.py

```diff
@@ -501,10 +507,41 @@ def extra_points_made(self):
         """
         return self._extra_points_made
 
+    @_int_property_decorator
+    def extra_points_attempted(self):
+        """
+        Returns an ``int`` of the total number of extra points the player
+        attempted.
+        """
+        return self._extra_points_attempted
+
+    @_float_property_decorator
+    def extra_point_percentage(self):
+        """
+        Returns a ``float`` of the percentage of extra points the player made.
```

Comment:
Looks like this still needs a note about the percentage range (0.0 - 100.0) as with the other attribute below.

Reply:
done!